pkg/certificate/providers/tresor/certificate_manager.go

```diff
@@ -22,7 +22,11 @@ func (cm *CertManager) issue(cn certificate.CommonName, validityPeriod time.Dura
 		return nil, errNoIssuingCA
 	}
 
-	certPrivKey, err := rsa.GenerateKey(rand.Reader, rsaBits)
+	certKeyBitSize := rsaBits
```

Comment:
Let's drop the default `rsaBits` here, and encode a default for this in `pkg/configurator` similar to `defaultServiceCertValidityDuration`.